x/reward/keeper/msg_server_test.go

```diff
@@ -0,0 +1,563 @@
+package keeper_test
+
+import (
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/provenance-io/provenance/x/reward/types"
+)
+
+func (s *KeeperTestSuite) TestCreateRewardProgramTransaction() {
+	s.SetupTest()
```

Comment:
I see this `s.SetupTest()` call in a lot of tests. That's being called by the `suite.Suite` stuff automatically. So calling it explicitly is calling it a second time.

Reply:
agreed